js/lib/src/state/matches.ts

```diff
@@ -33,6 +33,23 @@ export enum MatchState {
   Deactivated,
 }
 
+export function convertMatchState(ms: MatchState): {} {
```

Comment:
I don't see where you are actually calling this function. I do see you importing it into the contract, but I don't see it being used anywhere.

Reply:
Deleted this